### PR TITLE
FEATURE: SD-2933 Update workspaceDataFilters.json, workspaceDataFilterSettings.json, child workspace workspace.json

### PR DESCRIPTION
**Pull request by thuytran-gooddata**
…rSettings.json, child workspace workspace.json

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/162/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/171/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/163/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/172/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/164/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/173/ : SUCCESS in 6s

File: ecommerce-demo/workspaces/childDemo_bigboxretailer/workspaceSettings.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{
+  "colorPalette": {
+    "colorPaletteId": "demo_fresh",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_bigboxretailer/workspaceSettings.json:3]: Should we change `"demo_fresh"` → `"demo_colorPalette_fresh"` to be consistent with the naming for the theme id (at line 88), with has a value of `"demo_theme_xxx"`.

File: ecommerce-demo/workspaces/childDemo_clothing/workspaceSettings.json [hunk 2]
```diff
@@ -0,0 +1,178 @@
+{
+  "colorPalette": {
+    "colorPaletteId": "demo_modern",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_clothing/workspaceSettings.json:3]: Should we change `"demo_modern"` → `"demo_colorPalette_modern"` to be consistent with the naming for the theme id (at line 72), with has a value of `"demo_theme_xxx"`.

File: ecommerce-demo/workspaces/childDemo_electronics/workspaceSettings.json [hunk 3]
```diff
@@ -0,0 +1,204 @@
+{
+  "colorPalette": {
+    "colorPaletteId": "demo_contrast",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_electronics/workspaceSettings.json:3]: Should we change `"demo_contrast"` → `"demo_colorPalette_contrast"` to be consistent with the naming for the theme id (at line 88), with has a value of `"demo_theme_xxx"`.

File: ecommerce-demo/workspaces/childDemo_bigboxretailer/workspaceSettings.json (hunk 1 above)
Comment (thuytran-gooddata) [reply ecommerce-demo/workspaces/childDemo_bigboxretailer/workspaceSettings.json]: I updated it thanks

File: ecommerce-demo/workspaces/childDemo_bigboxretailer/workspace.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "data": {
+    "attributes": {
+      "name": "E-commerce Merchant Bigboxretailer"
+    },
+    "id": "ecommerce-merchant-bigboxretailer",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_bigboxretailer/workspace.json:6]: Should we add `demo_` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_ecommerce_merchant_bigboxretailer",
```

File: ecommerce-demo/workspaces/childDemo_bigboxretailer/workspaceDataFilterSettings.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
+{
+  "workspaceDataFilterSettings": [
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-bigboxretailer",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_bigboxretailer/workspaceDataFilterSettings.json:6]: Should we add `demo_wdfSettings` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_wdfSettings_ecommerce_merchant_bigboxretailer",
```

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/167/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/176/ : SUCCESS in 4s

File: ecommerce-demo/workspaces/childDemo_clothing/workspace.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+  "data": {
+    "attributes": {
+      "name": "3. Demo Workspace / Customer B (Child)"
+    },
+    "id": "ecommerce-merchant-clothing",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_clothing/workspace.json:6]: Should we add `demo_` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_ecommerce_merchant_clothing",
```

File: ecommerce-demo/workspaces/childDemo_clothing/workspaceDataFilterSettings.json [hunk 7]
```diff
@@ -0,0 +1,46 @@
+{
+  "workspaceDataFilterSettings": [
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-clothing",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_clothing/workspaceDataFilterSettings.json:6]: Should we add `demo_wdfSettings` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_wdfSettings_ecommerce_merchant_clothing",
```

File: ecommerce-demo/workspaces/childDemo_clothing/workspaceDataFilterSettings.json [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+              "type": "workspaceDataFilter"
+            }
+          }
+        }
+      }
+    },
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-clothing-cat",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_clothing/workspaceDataFilterSettings.json:26]: Should we add `demo_wdfSettings` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_wdfSettings_ecommerce_merchant_clothing_cat",
```

File: ecommerce-demo/workspaces/childDemo_bigboxretailer/workspaceDataFilterSettings.json [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+              "type": "workspaceDataFilter"
+            }
+          }
+        }
+      }
+    },
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-bigboxretailer-cat",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_bigboxretailer/workspaceDataFilterSettings.json:26]: Should we add `demo_wdfSettings` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_wdfSettings_ecommerce_merchant_bigboxretailer_cat",
```

File: ecommerce-demo/workspaces/childDemo_electronics/workspaceDataFilterSettings.json [hunk 10]
```diff
@@ -0,0 +1,45 @@
+{
+  "workspaceDataFilterSettings": [
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-electronics",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_electronics/workspaceDataFilterSettings.json:6]: Should we add `demo_wdfSettings` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_wdfSettings_ecommerce_merchant_electronics",
```

File: ecommerce-demo/workspaces/childDemo_electronics/workspaceDataFilterSettings.json [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+              "type": "workspaceDataFilter"
+            }
+          }
+        }
+      }
+    },
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-electronics-cat",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_electronics/workspaceDataFilterSettings.json:26]: Should we add `demo_wdfSettings` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_wdfSettings_ecommerce_merchant_electronics_cat",
```

File: ecommerce-demo/workspaces/childDemo_electronics/workspace.json [hunk 12]
```diff
@@ -0,0 +1,17 @@
+{
+  "data": {
+    "attributes": {
+      "name": "4. Demo Workspace / Customer C (Child)"
+    },
+    "id": "ecommerce-merchant-electronics",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/childDemo_electronics/workspace.json:6]: Should we add `demo_` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_ecommerce_merchant_electronics",
```

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/169/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/178/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/170/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/179/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/171/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/180/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/172/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/181/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/173/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/182/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/174/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/183/ : SUCCESS in 4s

**Comment by anvuong119** [comment]
LGTM

**Review by huavannis — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/175/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/184/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/176/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/185/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/177/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/186/ : SUCCESS in 5s

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-public-demos-gate-PR79

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/185/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-merge-compliance-check/67/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-sonarqube-scan-zuul-gate/65/ : SUCCESS in 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-fossa-scanning-zuul/28/ : FAILURE in 14s (non-voting)

**Pull request merged**